src/InstrumentationEngine.Preinstall/InstrumentationEngine.Preinstall.csproj

```diff
@@ -3,6 +3,7 @@
       -->
 <Project>
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), EnlistmentRoot.marker))\build\Common.props" />
+  <Import Project="$(EnlistmentRoot)\build\Packaging.props" />
   <Import Project="Sdk.props" Sdk="Microsoft.NET.Sdk" />
   <PropertyGroup>
```

Comment:
did you need to version bump?

Reply:
Luckily not for this change

Reply:
It appears this only fixes local build but not from ADO. Currently investigating...